docs/quick-start/view-data.txt

```diff
@@ -136,6 +136,31 @@ View MongoDB Data
 
          </body>
          </html>
+ 
+   .. _laravel-quick-start-json:
+
+   .. step:: Optionally, view your results as JSON documents
+
+      Rather than generating a view and editing the ``browse_movie.blade.php`` file, you can
+      use the ``toJson()`` method to display your results in JSON format.
+      
+      Replace the ``show()`` function with the following code to retrieve results and
+      return them as JSON documents:
+
+      .. code-block:: php
+
+         public function show()
+         {
+             $results = Movie::where('runtime', '<', 60)
+                 ->where('imdb.rating', '>', 8.5)
+                 ->orderBy('imdb.rating', 'desc')
+                 ->take(10)
+                 ->get();
+
+             foreach ($results as $movie) {
+                 echo $movie->toJson() . '<br>\n';
+             }
```

Comment:
Using `echo` in a controller is considered a bad practice as it skips the `Response` object. It makes it impossible to send HTTP headers.

`$results` is an instance of `Illuminate\Database\Eloquent\Collection`.

```suggestion
             return $results->toJson();
```